ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -1210,6 +1219,26 @@ extension BottomCommandingController: CommandingItemDelegate {
             break
         }
     }
+
+    func commandingItem(_ item: CommandingItem, didChangeHiddenTo value: Bool) {
+        guard let view = itemToBindingMap[item]?.view else {
+            return
+        }
+
+        switch view {
+        case let tabBarItemView as TabBarItemView:
+            if tabBarItemView.isHidden != value {
```

Comment:
is this check needed? Shouldn't the no-op if value isn't changing be an implementation detail?